src/View/mainpage/DrawROIWindow/UIDrawROIWindow.py

```diff
@@ -403,22 +393,19 @@ def init_layout(self):
         self.draw_roi_window_transect_draw_box. \
             addWidget(self.image_slice_number_box_draw_button)
 
-        # Create a draw button
-        self.image_slice_number_draw_button = QPushButton()
-        self.image_slice_number_draw_button. \
-            setObjectName("ImageSliceNumberDrawButton")
-        self.image_slice_number_draw_button.setSizePolicy(
+        # Create a fill button
+        self.image_slice_number_fill_button = QPushButton()
+        self.image_slice_number_fill_button. \
+            setObjectName("ImageSliceNumberFillButton")
+        self.image_slice_number_fill_button.setSizePolicy(
             QSizePolicy(QSizePolicy.MinimumExpanding, QSizePolicy.Minimum))
-        self.image_slice_number_draw_button.resize(
-            self.image_slice_number_draw_button.sizeHint().width(),
-            self.image_slice_number_draw_button.sizeHint().height())
-        self.image_slice_number_draw_button.clicked.connect(self.onDrawClicked)
-        icon_draw = QtGui.QIcon()
-        icon_draw.addPixmap(QtGui.QPixmap(
-            resource_path('res/images/btn-icons/draw_icon.png')))
-        self.image_slice_number_draw_button.setIcon(icon_draw)
+        self.image_slice_number_fill_button.resize(
+            self.image_slice_number_fill_button.sizeHint().width(),
+            self.image_slice_number_fill_button.sizeHint().height())
+        self.image_slice_number_fill_button.clicked.connect(self.onFillClicked)
+        # TODO: Add fill icon
```

Comment:
need a  decision on this

Reply:
Pedro, do you mind clarifying this comment, please? I do not wish to accept the PR unless the reviewers are happy for me to do so.

Reply:
Generally TODOs have good intention, but get left indefinitely in code. Either it needs to be removed, with the decision of not having an icon, or a story needs to be created to keep track of the todo at a later date.

This Todo is small so its okay, will have to make a story just to confirm it gets done.